### ISSUE TITLE
Mongoose internal types not serialized

### ISSUE THREAD

**Issue by mcaldas** [ForestAdmin/forest-express-mongoose#78]
composite mongoose types aren't serialized, for example `mongoose.Schema.Types.Point`

**Comment by mcaldas** [comment]
@arnaudbesnier

**Comment by arnaudbesnier** [comment]
Yes we have this issue in mind. I'll let you know if we can add this in the next sprint.

**Comment by lucasscariot** [comment]
Are you using `mongoose-geojson-schema` for the `mongoose.Schema.Types.Point` ?
And can you send me an example of your model to help me reproduce the bug?

**Comment by mcaldas** [comment]
Hey!
I shared an example model in #76 , is that enough?
For geojson yes, we use `mongoose.Schema.Types.Point`

**Comment by mcaldas** [comment]
@lucasscariot :)

**Comment by arnaudbesnier** [comment]
@mcaldas, I am not sure to understand.

Tell me if I am wrong but, according to the doc, `mongoose.Schema.Types.Point` is not a "native" Mongoose type.
What you actually is Forest to support the `mongoose-geojson-schema`, right?

**Comment by arnaudbesnier** [comment]
@mcaldas do you still need some help?
We didn't receive any response since my last question 3 month ago.
